### ISSUE TITLE
Make calledWith report what it wasn't called with

### ISSUE THREAD

**Issue by jmcdo29** [43081j/hanbi#11]
Yo, first off been loving this little library, amazingly easy to set up and use, very lightweight! Absolutely love it! :heart: :rocket: 

Would it be possible to write out to `process.stdout` or `process.stderr` when `calledWith` fails? Something like `Expected ${this.original.name} spy to be called with ${args} but it never was`? Right now I'm stepping into `spy.getCall(n).args[k]` and asserting the value with `uvu` because I get a better error message, but having an immediate write out would be very useful.

**Comment by 43081j** [comment]
I want to avoid building assertion functions and errors into this to be honest. The idea is that you would use this in conjunction with an assertion library (like the one uvu ships). 

Could you achieve the same by just asserting that calledWith was truthy?

Is the problem with that, that you want nicer errors? 

It could end up being more that we need better integration with popular tools rather than changing the core. Once I understand more, hopefully I'll have some ideas

**Comment by jmcdo29** [comment]
Yeah, I'm realizing I could set the message directly in `uvu`'s `ok` or similar and adding this to `hanbi` would be overhead that could make other integrations more difficult. Right now, I can easily just `ok(spy.calledWith(parameters))`, but when that fails there's no nice error message, just that this comes back as false. But I can set up pretty much the same error message in my original comment here and get the same functionality through a higher order function.

Thanks again for the library!
